package/android/build.gradle

```
@@ -31,6 +31,7 @@ def getExtOrIntegerDefault(name) {
 }
 
 android {
+  namespace 'com.reactnativecommunity.slider'
```

Might be better to check for the agp version
```suggestion
    def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
    if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
        namespace "com.reactnativecommunity.slider"
    }
```

---

It's only available from 7.3+, not 7, so this check will fail if user is on an older version.